### ISSUE TITLE
Aphid Maze fails to load in Ladybug-motion-2D

### ISSUE THREAD

**Issue by loganbraywork** [phetsims/decaf#24]
For https://github.com/phetsims/QA/issues/489
https://phet-dev.colorado.edu/decaf/ladybug-motion-2d/

Issue appears on FireFox.
The aphid maze sim in Ladybug-motion-2D fails to load and reads as "null". The ladybug motion 2D sim loads successfully.
![2020-04-01DecafLadybug-in-motion-2D](https://user-images.githubusercontent.com/54685448/78190933-3be0da80-7432-11ea-97b5-62e4e6b21a74.gif)

**Comment by KatieWoe** [comment]
Likely same as https://github.com/phetsims/decaf/issues/9

**Comment by samreid** [comment]
I don't believe "Aphid Maze" is a production sim that we wish to port.  @kathy-phet or @KatieWoe please correct me and reopen if incorrect.

**Comment by KatieWoe** [comment]
I don't believe it is, but it that case, shouldn't it be removed as a selectable option @samreid?

**Comment by samreid** [comment]
Sounds like a plan, I'll do that, thanks!
